nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/test/java/org/nuxeo/ecm/automation/core/operations/security/PermissionAutomationTest.java

```diff
@@ -87,14 +101,26 @@ public void initRepo() throws Exception {
 
         when(administratorGroupsProvider.getAdministratorsGroups()).thenReturn(
                 Collections.singletonList("administrators"));
+
+        DocumentModel testUser = userManager.getBareUserModel();
```

Comment:
In our case, what are the benefices to mock instead using real test preparation? We can have less dependencies when using mock, but `PlatformService` is in the `test` scope, so it should be alright, isn't?

Obviously, only the User ID `testUserId` is stored here, but it doesn't mean it's the only thing we use: A real user need to be created before each test. If the target user `existingUser ` does not exist, all the tests based on this user will fail. Such failure is raised by the user-existence-check in `AddPermission.java`, added in this ticket.